api/v1/openapi.yaml

```diff
@@ -944,7 +944,7 @@ definitions:
       build:
         description: Build number of calculated policy in use
         type: integer
-      allowed-consumers:
+      allowed-ingress-identities:
```

Comment:
Is this whole object marked in some way as "may change without notice"? It's leaking internal policy structures that I expect will change in subsequent Cilium version.

Reply:
I'm not entirely sure what you mean by this -   the API is not yet stable since we have not officially released v1.0.

Reply:
IMHO most of the fields in the endpoint here are implementation details which we should be free to change. I don't really think it should be formalised in the API, but if it is then I'm proposing that we mark it as an unstable portion of the API, so people shouldn't expect to rely on it.

Reply:
I agree about marking this portion of the API as unstable. However, I am of the belief that we need to have clear use of our vocabulary throughout all parts of the Cilium - it is not clear to a user of the API was a 'consumer' is, but 'identity' is used throughout.

Reply:
Agree, 'consumer' isn't mentioned in the docs at all but 'identity' is described in the Concepts section.